plugins/database/mysql-ha/src/main/java/com/cloud/utils/db/StaticStrategy.java

```diff
@@ -44,21 +44,21 @@ public JdbcConnection pickConnection(InvocationHandler proxy, List<String> confi
 
         SQLException ex = null;
 
-        List<String> whiteList = new ArrayList<String>(numHosts);
-        whiteList.addAll(configuredHosts);
+        List<String> allowList = new ArrayList<String>(numHosts);
+        allowList.addAll(configuredHosts);
 
-        Map<String, Long> blackList = ((LoadBalancedConnectionProxy) proxy).getGlobalBlacklist();
+        Map<String, Long> denylist = ((LoadBalancedConnectionProxy) proxy).getGlobalBlacklist();
```

Comment:
maybe good to change that method name as well? `getGlobalBlacklist()` ==>> `getGlobalDenylist()`